crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1364,20 +1424,15 @@ impl Client {
                 self.server_versions().await?,
                 send_progress,
             )
-            .await;
-
-        if let Err(http_error) = &response {
-            if let Some(ErrorKind::UnknownToken { soft_logout }) =
```

Comment:
This I why you get the ErrorKind error, no it's only above, in the method behind the `experimental-sliding-sync` cargo feature, so the import must be too.

Reply:
Ah ha! I see it now 😁